src/app/pages/HomePage/index.tsx

```diff
@@ -139,7 +139,7 @@ export const HomePage: FC = () => {
           </LogotypeBox>
           <SearchInputContainer>
             <SearchInputBox>
-              <Search disabled={isApiOffline} variant={searchVariant} onFocusChange={onFocusChange} />
+              <Search disabled={!isApiReachable} variant={searchVariant} onFocusChange={onFocusChange} />
```

Comment:
I think we need to update this to get the inside from within the structure returned by the hook.

Reply:
:scream: